### PR TITLE
Rename extension, update dependencies

### PR DESCRIPTION
**Pull request by sfoslund**
- Renaming extension to match new repo name
- Updating dependencies per [security alert](https://github.com/dotnet/vscode-dotnet-runtime/network/alert/dotnetcore-acquisition-library/package-lock.json/diff/open)
- Refactoring file structure in acquisition-library

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ pr:
 
 variables:
   is-release: $[startsWith(variables['Build.SourceBranch'], 'refs/tags/v')]
-  package-name: 'dotnetcore-acquisition'
+  package-name: 'vscode-dotnet-runtime'
```
Comment (NTaylorMullen) [inline azure-pipelines.yml]: 👏

File: dotnetcore-acquisition-extension/package.json [hunk 2]
```diff
@@ -1,16 +1,16 @@
 {
-	"name": "dotnetcore-acquisition",
+	"name": "vscode-dotnet-runtime",
```
Comment (NTaylorMullen) [inline dotnetcore-acquisition-extension/package.json]: Any reason you're not also changing the folder name `dotnetcore-acquisition-extension` -> `vscode-dotnet-runtime-extension`

Comment (sfoslund) [reply dotnetcore-acquisition-extension/package.json]: Not in particular, I figure that it's not exposed externally so might as well keep it the same. I can update it if you think that's more clear.

Comment (NTaylorMullen) [reply dotnetcore-acquisition-extension/package.json]: Would probably make it more clear to have the repo consistent but not a big deal either 😄

Comment (sfoslund) [reply dotnetcore-acquisition-extension/package.json]: Latest commit has the folder names updated 👍

**Review by LLotme — COMMENTED** [review]
gh pr checkout 37

**Pull request merged**